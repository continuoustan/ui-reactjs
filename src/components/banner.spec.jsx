import React from 'react';
import TestUtils from 'react-dom/test-utils';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';

import Banner from './banner';


test('Banner renders correctly', () => {
  const tree = renderer.create(
    <Banner>
      Text inside
    </Banner>
  );

  expect(tree).toMatchSnapshot();
});

