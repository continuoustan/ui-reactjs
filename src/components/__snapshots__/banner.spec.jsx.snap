// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner renders correctly 1`] = `
<div
  id="banner"
>
  <div
    className="wfp-wrapper"
  >
    <div
      className="wfp-hidden-lg wfp-hidden-xl"
    >
      Text inside
    </div>
    <div
      className="wfp-hidden-md wfp-hidden-xs"
    >
      Text inside
    </div>
  </div>
</div>
`;
